config/stage.json

```diff
@@ -49,6 +49,7 @@
 		"manage/security": true,
 		"manage/site-settings/analytics": true,
 		"manage/site-settings/categories": true,
+		"manage/site-settings/disconnect-flow": false,
```

Comment:
Is it working? If it’s not it would be confusing for proxied Automatticians using WordPress.com.